validate_cmd.go

```diff
@@ -18,26 +23,59 @@ var validateCmd = &cobra.Command{
 	RunE:  runValidate,
 }
 
+type LintMessage struct {
+	Name        string `json:"name"`
+	Code        string `json:"code"`
+	Severity    string `json:"severity"`
+	Path        string `json:"path"`
+	Line        int    `json:"line"`
+	Char        int    `json:"char"`
+	Description string `json:"description"`
+}
+
+// Builds a Json file with the issues found after parsing the requirements and code. It only collects
+// information for the base repository.
+// @llr REQ-TRAQ-SWL-62
+func buildJsonIssues(issues []Issue, jsonWriter *json.Encoder) {
+	for _, issue := range issues {
+		// Only report issues for the current repository
+		if issue.RepoName != repos.BaseRepoName() {
+			continue
+		}
+
+		jsonWriter.Encode(LintMessage{
+			Name:        "Reqtraq",
+			Code:        "Reqtraq",
```

Comment:
Why all errors with same name? It'll be ugly in phabricator

Reply:
As far as I understand we have the same for clang-format:
"name":"code must be clang-formatted",

I'm open to suggestions about what to use instead, but the error will be in the message